package.json

```diff
@@ -64,6 +64,7 @@
     "@bbc/psammead-brand": "^5.1.11",
     "@bbc/psammead-bulleted-list": "1.0.5",
     "@bbc/psammead-bulletin": "^1.0.7",
+    "@bbc/psammead-byline": "^0.1.0-alpha.1",
```

Comment:
Needs to come out of alpha before we can merge: https://github.com/bbc/simorgh/issues/5496